### PR TITLE
Remove handlebars-utils package

### PR DESCRIPTION
**Pull request by adrinr**
Remove handlebars-utils package. The newly introduced code comes from the removed package, being copied in here.
The original repo is making use of a lot of small packages that depend on each other. As we want to migrate to ES, we have 2 options:
1. Fork, use and migrate all the projects
2. Copy the code that is actually required into this project

Given that the packages are quite small and most of them don't do much, we decided to copy the code over. This will also allow us refactoring and cleaning the code much easier.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -3221,7 +3221,7 @@ gulplog@^1.0.0:
   dependencies:
     glogg "^1.0.0"
 
-handlebars-utils@^1.0.4, handlebars-utils@^1.0.6:
+handlebars-utils@^1.0.4:
```
Comment (samwho) [inline yarn.lock:3224]: Should this not entirely be gone now?

**Review by samwho — COMMENTED** [review]
For historical purposes could you detail the rationale for this change in the PR description? 🙏

File: yarn.lock (hunk 1 above)
Comment (adrinr) [reply yarn.lock:3224]: Due the maze of dependencies, this is still used for a dev package called "verb". This should probably not be needed, but given that is a dev dependency we should be good for now

**Comment by adrinr** [comment]
> For historical purposes could you detail the rationale for this change in the PR description? 🙏

Good shout! Done

**Pull request merged**